src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -695,6 +696,7 @@ void Transformations::PreLpt(const std::vector<ov::element::Type>& defaultPrecis
     CPU_DISABLE_PASS_COMMON(manager, ov::pass::MatMulConstTransposesExtraction);
     CPU_DISABLE_PASS_COMMON(manager, ov::pass::ConvertScatterNDUpdate15ToScatterNDUpdate3);
     CPU_DISABLE_PASS_COMMON(manager, ov::pass::ConvertSliceScatter);
+    CPU_DISABLE_PASS_COMMON(manager, ov::pass::SDPAFusion);
```

Comment:
Why was it disabled for CPU?
It's quite strange because the transformation is enabled in MOC. It means SDPA op will be created in IR but if we use the runtime path 
`read_model(fw_model) + compile`  on CPU  SDPA won't be generated.

Reply:
Because CPU has tests that runs CPU plugin twice for different input models (w/ and w/o SDPA op). If we enable this pass for CPU such tests fail. So CPU plugin team need to rewrite such test cases first to use ngraph reference (or any other way to compute ref), and then they can enable this pass